source/standalone/workflows/skrl/play.py

```diff
@@ -0,0 +1,204 @@
+        device=env.device,
+    )
+
+    # specify directory for logging experiments (load checkpoint)
+    print(f"[INFO] Loading experiment from directory: {log_checkpoint_path}")
+    # get checkpoint path
+    if args_cli.checkpoint:
+        resume_path = os.path.abspath(args_cli.checkpoint)
+    else:
+        resume_path = get_checkpoint_path(log_checkpoint_path, os.path.join("*", "checkpoints"), None)
+    print(f"[INFO] Loading model checkpoint from: {resume_path}")
+    agent.load(resume_path)
+
+    # test the agent according to the selected mode defined with "use_api":
+    # - True: a skrl trainer will be used to evaluate the agent
+    # - False: the interaction with the environment will be performed manually.
+    #          This mode allows recording specific information about the environment
+
+    # configure and instantiate the RL trainer
+    # https://skrl.readthedocs.io/en/latest/modules/skrl.trainers.sequential.html
+    if experiment_cfg["trainer"]["use_api"]:
+        trainer_cfg = experiment_cfg["trainer"]
+        trainer_cfg["disable_progressbar"] = True
+        trainer = ManualTrainer(cfg=trainer_cfg, env=env, agents=agent)
+
+        # simulate environment
+        while simulation_app.is_running():
+            # agent and environment stepping
+            trainer.eval()
+            # check if simulator is stopped
+            if env.sim.is_stopped():
+                break
+
+    # execute the interaction with the environment without using a trainer
+    # (note that skrl requires the execution of some additional agent methods for proper operation).
+    # https://skrl.readthedocs.io/en/latest/modules/skrl.trainers.manual.html
+    else:
+        timestep = 0
+        agent.init()
+        agent.set_running_mode("eval")
+
+        # reset environment
+        obs, infos = env.reset()
+        # simulate environment
+        while simulation_app.is_running():
+            timestep += 1
+            # agent stepping
+            actions = agent.act(obs, timestep=timestep, timesteps=None)[0]
+            # env stepping
+            next_obs, rewards, terminated, truncated, infos = env.step(actions)
+            # track data
+            agent.record_transition(
+                states=obs,
+                actions=actions,
+                rewards=rewards,
+                next_states=next_obs,
+                terminated=terminated,
+                truncated=truncated,
+                infos=infos,
+                timestep=timestep,
+                timesteps=None,
+            )
+            # log custom environment data
+            if "episode" in infos:
+                for k, v in infos["episode"].items():
+                    if isinstance(v, torch.Tensor) and v.numel() == 1:
+                        agent.track_data(f"Info / {k}", v.item())
+            # write data to TensorBoard / Weights & Biases
+            super(type(agent), agent).post_interaction(timestep=timestep, timesteps=None)
```

Comment:
Do we need logging when playing?

Reply:
No... I just find it useful. But it can be removed without any problem